### ISSUE TITLE
Revamp the settings activity to be cleaner & more elegant (Priority: high)

### ISSUE THREAD

**Issue by jchio001** [kkong006/CS180FrontEnd#9]
MSPaint-y picture of what I want:

[http://imgur.com/a/PsVGQ](http://imgur.com/a/PsVGQ)

Note: Because there's not that many elements, a listview is not needed and you can just hardcode each element. Also because each (future) settings element will look different from each other and won't follow an easily recyclable pattern, hardcoding's probably better.

**Comment by jchio001** [comment]
Swapped out the dropdown with an AutoCompleteTextView because scrolling through dropdowns suck.

**Comment by jchio001** [comment]
Layout overhaul is done; however the API calls being made are failing on me & EventBus listeners are not picking up on any events.

tl;dr API calls broken, fix it.

**Comment by jchio001** [comment]
API calls seem to be working. Leaving ticket open since stuff still might break.

**Comment by jchio001** [comment]
I'll assume everything's working for now.
